core/src/test/kotlin/com/segment/analytics/kotlin/core/TelemetryTest.kt

```diff
@@ -89,23 +89,23 @@ class TelemetryTest {
     fun `Test increment when telemetry is disabled`() {
         Telemetry.enable = false
         Telemetry.start()
-        Telemetry.increment(Telemetry.INVOKE_METRIC, mapOf("test" to "test"))
+        Telemetry.increment(Telemetry.INVOKE_METRIC) { it["test"] = "test" }
         assertEquals(0, TelemetryQueueSize())
         assertEquals(0,errors.size)
     }
 
     @Test
     fun `Test increment with wrong metric`() {
         Telemetry.start()
-        Telemetry.increment("wrong_metric", mapOf("test" to "test"))
+        Telemetry.increment("wrong_metric") { it["test"] = "test" }
         assertEquals(0, TelemetryQueueSize())
         assertEquals(0,errors.size)
     }
 
     @Test
     fun `Test increment with no tags`() {
         Telemetry.start()
-        Telemetry.increment(Telemetry.INVOKE_METRIC, emptyMap())
+        Telemetry.increment(Telemetry.INVOKE_METRIC) { it.clear() }
```

Comment:
no need to clear. can just leave it with empty brackets. or make the `tagBuilder` nullable, so you don't have to pass anything

Reply:
It was in the test so I just figured I'd make it explicit about what it was expected to do, rather than minimize code.